crates/sui/src/client_commands.rs

```diff
@@ -3098,3 +3107,66 @@ async fn check_protocol_version_and_warn(client: &SuiClient) -> Result<(), anyho
 
     Ok(())
 }
+
+/// Fetch move packages based on the provided package IDs.
+async fn fetch_move_packages(
+    client: &SuiClient,
+    package_ids: &[ObjectID],
+) -> Result<Vec<MovePackage>, anyhow::Error> {
+    let objects = client
+        .read_api()
```

Comment:
It seems like you only ever use the `ReadApi`, so the wider changes to expose the `SuiClient` instead don't seem necessary?